Services/CompanyService/CompanyService.csproj

```diff
@@ -6,13 +6,4 @@
     <RootNamespace>Studfolio.CompanyService</RootNamespace>
   </PropertyGroup>
 
-  <ItemGroup>
-    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="3.1.8" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="3.1.8" />
```

Comment:
You can't change files from another services. Also, don't forget to merge develop into your branch sometimes so that your branch can include new code from develop